service/k8sclient/k8sclient_test.go

```diff
@@ -884,8 +973,12 @@ func TestVMAgentSpec(t *testing.T) {
 	assert.Equal(t, expected, inBuf.String())
 }
 
-func TestGetClusterState(t *testing.T) {
-	// t.Parallel()
+func TestGetPXCClusterState(t *testing.T) {
+	t.Parallel()
+	perconaTestOperator := os.Getenv("PERCONA_TEST_DBAAS_OPERATOR")
+	if perconaTestOperator != "pxc" {
```

Comment:
I think we should improve documentation to explain what this environment variable is about. Because with these changes if the user runs tests as they used to run no cluster-related tests will be run.

Reply:
I implemented backward compatibility with that changes. Now it uses "pxc", "psmdb" just to test previous version, however all tests run against recommended one.